src/main/java/com/commercetools/project/sync/util/ClientConfigurationUtils.java

```diff
@@ -37,13 +42,31 @@
    */
   public static SphereClient createClient(@Nonnull final SphereClientConfig clientConfig) {
 
-    final HttpClient httpClient = getHttpClient();
+    //final HttpClient httpClient = getHttpClient();
+    final HttpClient httpClient = createCustomHttpClient();
     final SphereAccessTokenSupplier tokenSupplier =
         SphereAccessTokenSupplier.ofAutoRefresh(clientConfig, httpClient, false);
     final SphereClient underlying = SphereClient.of(clientConfig, httpClient, tokenSupplier);
     final SphereClient retryClient = withRetry(underlying);
     return withLimitedParallelRequests(retryClient);
   }
+  
+  /**
+   * Creates a {@link HttpClient} that can be used by the {@link SphereClient}.
+   * @return new http client with custom settings
+   */
+  public static HttpClient createCustomHttpClient() {
+      ProxyServer proxy = new ProxyServer("one.proxy.att.com", 8080, 8080, null, null);
```

Comment:
This should not be hard-coded.  Usually proxies are defined via the env vars `http_proxy` and `https_proxy`